README.md

```diff
@@ -38,14 +38,14 @@ This website was created with [Docusaurus](https://v2.docusaurus.io).
 
 ```sh
 # Install dependencies
-$ npm install
+$ npm i
```

Comment:
I realize this is a bit of a nitpick but I'd rather keep install as a full word to be easier to learn

Reply:
Good point, I will put that back